docs/tutorial.md

```diff
@@ -0,0 +1,93 @@
+
+Before calling this function I need to approve Currency Factory to use my CLN tokens. It's like saying to CLN contract: "Hey contract, if someone named CurrencyFactory will try to use my tokens, it's ok, I approve this". This is a ERC20 mechanism.
+
+ Remeber how we called `createIssuance`? Now we'll do the same with the `ColuLocalNetwork` token. After uploading the ABI I'll call approve function with 1000 * 10 ^ 18 = 1000000000000000000000 (1000 CLN) as value and `CurrencyFactory` address as the spender. I checked everything is correct and click send.
+
+ After the approve transaction is confirmed I can eventually call `insertCLNtoMarketMaker`. I do the same process again, this time for `CurrencyFactory`, after ABI is loaded I select `insertCLNtoMarketMaker` function. There's actually two functions with the same name (this is Solidity's function overloading feature), and I need the one that received both `token` and `clnAmount`. Maybe we talk latter about the second one. In the token field I paste the `LeonCoin`'s address, and put the same 1000 * 10 ^ 18 for the ammount field.
+
+Viewing the [transaction](https://ropsten.etherscan.io/tx/0xaadef80fdbb2dc223a0c780e2d4444b0a8ec9642a1496970653f681bfa73c966) you may think some complex stuff happened there. Well, I'll try to sum it up.
+
+- `CurencyFactory` transfers the CLN to itself. He can do it cause I gave him an allowance for this before.
+- Then he transfers the tokens to Marker Maker letting him do his mathematical magic. More techically speaking he approves `MM` to use `CLN`, and calls `MM`'s  `change` function.
+- In return the `CurencyFactory` receives `LC` tokens.
+- He sends the tokens back to the sender.
+
+As you see, `Currency Factory` It's really just an intermediary that calls `Marker Maker` and holds some data about issuances. Now I'll add my `LC` token to MyEtherWallet, so I can easily see my tokens. I encourage you to do this with your Community Currency also, just for fun as I say.
+
+You can call `insertCLNtoMarketMaker` multiple times, exchanging more CLN for your home-baked Community Currency. By the way did you noticed that the CLN/CC exchange rate changes? This is because as you change more CLN to CC, your CC becomes more valuable and so It's price grows. That's how Market Maker works, we'll get into it in the second part of the tutorial.
+
+You call `extractCLNfromMarketMaker` to get your CLN back. If you exhcange all you CC to CLN you'll get the innitial CLN ammount, no CLN lost.
+
+Only token issuer can call `insertCLNtoMarketMaker` and `extractCLNtoMarketMaker`. For everyone else to be able to exchange CLN/CC we need to open the CC's Market Maker for public usage. Why this is done? So the issuer will have sufficient time for various prepations to make the new currency usable, like inserting enough CLN or even do something not related to crypto.. Then when he's ready he releases the new currency to the world.
```

Comment:
maybe add two sentences about that, the issuer can't create CC from thin air, he needs to pay them with CLN, also the closed market maker giving the issuer the advantage of acquiring the first CCs (the cheapest).
This means that the cc automatically has real value.